index.html

```diff
@@ -136,38 +228,105 @@ <h4>Don't Forget These!</h4>
+          <input type="email" class="form-control" id="validationCustom02" placeholder="email@website.com"  required>
+          <div class="valid-feedback">
+                  At least it is not a hotmail account...
           </div>
-          <div class="form-check">
-            <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2" value="option2">
-            <label class="form-check-label" for="exampleRadios2">
-              Mountain
-            </label>
+          <div class="invalid-feedback">
+                    That's not a real email!
           </div>
-          <button type="submit" class="btn btn-info">Let's Travel</button>
-        </form>
+        </div>
+
+        <div class="mt-2 row radios">
+
+            <div class=" col-3 form-check">
+              <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
+              <label class="form-check-label" for="exampleRadios1">
+                Beach
+              </label>
+            </div>
+
+            <div class=" col-3 form-check">
+              <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2" value="option2">
+              <label class="form-check-label" for="exampleRadios2">
+                Mountain
+              </label>
+            </div>
+
+            <div class="col-3 form-check">
+              <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios2" value="option3">
+              <label class="form-check-label" for="exampleRadios2">
+                Island
+              </label>
+            </div>
+
+            <button type="submit" class=" col-3 btn btn-info">Let's Travel</button>
+      </div>
+
       </div>
+
+    </form>
+
+    <script>
+    // Example starter JavaScript for disabling form submissions if there are invalid fields
+    (function() {
+      'use strict';
+      window.addEventListener('load', function() {
+        // Fetch all the forms we want to apply custom Bootstrap validation styles to
+        var forms = document.getElementsByClassName('needs-validation');
+        // Loop over them and prevent submission
+        var validation = Array.prototype.filter.call(forms, function(form) {
+          form.addEventListener('submit', function(event) {
+            if (form.checkValidity() === false) {
+              event.preventDefault();
+              event.stopPropagation();
+            }
+            form.classList.add('was-validated');
+          }, false);
+        });
+      }, false);
+    })();
+    </script>
     </div>
+
     <div class="row footer">
-      <div class="col-sm-12">
+      <div class="copyright col-3">
         Copyright Fun Bus 2018
       </div>
     </div>
+
   </div><!-- container -->
 
     <script src="js/jquery-3.3.1.min.js"></script>
     <script src="js/bootstrap.bundle.min.js"></script>
     <script src="js/index.js"></script>
```

Comment:
Line 330: 

To get the tooltip working you can add:
```
<script>
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
</script>
```
_after_ line 330 and the tooltip should work.